ElectronClient/app/gui/NoteList.jsx

```diff
@@ -293,20 +293,43 @@ class NoteListComponent extends React.Component {
 		}
 	}
 
+	scrollNoteIndex_(keyCode, ctrlKey, metaKey, noteIndex) {
+
+		if (keyCode === 33) {
+			noteIndex -= (this.itemListRef.current.visibleItemCount() - 1);
+		}
+		if (keyCode === 34) {
+			noteIndex += (this.itemListRef.current.visibleItemCount() - 1);
+		}
+		if ((keyCode === 35 && ctrlKey) || (keyCode === 40 && metaKey)) {
+			noteIndex = this.props.notes.length - 1;
+		}
+		if ((keyCode === 36 && ctrlKey) || (keyCode === 38 && metaKey)) {
+			noteIndex = 0;
+		}
+		if (keyCode === 38 && !metaKey) {
+			noteIndex -= 1;
+		}
+		if (keyCode === 40 && !metaKey) {
+			noteIndex += 1;
+		}
```

Comment:
Please add the name of the key in a comment, next to the keycode as this make it easier to see what is what.

Reply:
Done